### ISSUE TITLE
Support 'const' keyword

### ISSUE THREAD

**Issue by InDIOS** [NectarJS/nectarjs#104]
Hi, great work with this amazing tool. I've tested some examples and some self code and i've seen that you support 'let' beside 'var' keyword. Have you any plans to support 'const' keyword soon? I've made some tests and is not very complicated, i'm not a C++ dev but i know that it support const variables and this is my suggestion.

You use babel as js parser and in code genration process of ast there is no check for `VariableDeclaration` kind property, it means that you can put whatever you want as kind and it will be generated.

This is the test:
`compiler/native/src/macro.h`
```c
#define __NJS_CONST const NJS::VAR
```
and in
`compiler/native/visitor/plugin/VariableDeclaration.js`
```js
if (_path.node.declarations) {
    if (_path.node.kind === 'const') {
      _path.node.kind = '__NJS_CONST';
    }
    ...
```
I tested this solution and it works very well. You could, of course, use eslint to check constant reasigments but, this is just a suggestion.
Again great work, i'm glad that somebody put your effors in somthing like this idea.

**Comment by adrien-thierry** [comment]
Hi @InDIOS ,

Thank you very much for your message and for your tip and test, i will implement it right now.

**Comment by adrien-thierry** [comment]
Added in 2f2759d
and published in v0.7.56

Thx !
